VideoAPI/VideoApi.Services/AudioPlatformService.cs

```diff
@@ -277,6 +232,14 @@ private async Task CreateAndUpdateApplicationAsync(string applicationName)
             _logger.LogInformation($"Updating Wowza application for: {applicationName}");
         }
 
+        private void LogError(AudioPlatformException ex, string errorMessage)
```

Comment:
if you make the response type `AudioPlatformServiceResponse` then you can return the method call rather than logging the error and then creating the model

Reply:
Good point, but some of the methods return the `AudioPlatformServiceResponse` with more properties set, e.g. IngestUrl = DefaultEmptyIngestUrl. I can pass default params in but will get a bit messy. Better to be explicit and see what the method is returning clearly.